devito/dse/aliases.py

```diff
@@ -239,17 +239,17 @@ class Alias(object):
     is tracked.
     """
 
-    def __init__(self, alias, dimensions, aliased=None, distances=None,
+    def __init__(self, alias, indices, aliased=None, distances=None,
                  ghost_offsets=None):
         self.alias = alias
-        self.dimensions = dimensions
+        self.dimensions = tuple(i.parent if i.is_Derived else i for i in indices)
```

Comment:
yes, I screwed names; in fact, these should be called ``dimensions``, while sometimes (in this file) I used ``indices`` instead of ``dimensions``; fixing.